### PR TITLE
Remove whitespace around codeblock fences 

### PR DESCRIPTION
**Pull request by jason-fox**
Fix for #310

### CODE REVIEW

File: specs/howto.md [hunk 1]
```diff
@@ -316,7 +316,7 @@ Examples of GeoJSON geometries are available
   [UN/CEFACT](https://www.unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_rev3_Annex3e.pdf). 
   E.g.:
 
-  ```json
+```json
```
Comment (chicco785) [inline specs/howto.md]: so readthedocs does not support indentation for code blocks to align them with bullets... a pity

on the other side, i think we should add a short guide on how to local generate the readthedocs so to check that they work fine

**Comment by chicco785** [comment]
ps: also this list is not rendered correctly: https://github.com/Fiware/dataModels/pull/311/files#diff-9308877b8fc4a861aa6d718690d52f82R72

**Comment by jason-fox** [comment]
We could either manually add two spaces on the sub list or push #312 and never have the problem again. I would be happy to abandon this if the other PR is accepted.

**Comment by jason-fox** [comment]
Withdrawn - fixed by #312

**Pull request closed without merging**